settings.py

```diff
@@ -55,6 +58,13 @@
     'submissions'
 )
 
+# TODO: These are removed from global defaults. Not sure we need here or not but i have added this to remove warning.
```

Comment:
These middleware were removed from defaults. I'm leaving them as it without the TODO otherwise it will show warnings.